lib/arclight/custom_document.rb

```diff
@@ -31,7 +31,7 @@ class CustomDocument < SolrEad::Document
     def to_solr(solr_doc = {})
       super
       solr_doc['id'] = Arclight::NormalizedId.new(eadid.first).to_s
-      arclight_field_definitions.each do |field|
+      [*arclight_field_definitions, *arclight_field_definitions_colls].each do |field|
```

Comment:
Is there any particular reason that we're splitting these field definitions into two separate arrays?  I'm not really following what about the `_colls` makes this array different than `arclight_field_definitions`.